scripts/deploy/bump-version.js

```diff
@@ -32,10 +33,19 @@ async function checkoutLatestMaster() {
   await exec('git pull origin master');
 }
 
+/**
+ * @returns {Promise<string[]>}
+ */
+async function getPackagesToBump() {
+  const {stdout} = await exec('git ls-files | grep -e "/package.json$"');
+  const packages = stdout.trim().split('\n').map(filePath => JSON.parse(readFileSync(filePath).toString()));
+  return packages.filter((npmPackage) => npmPackage.version && !npmPackage.private).map(({name}) => name)
+}
+
 async function bumpVersionAndPush() {
   try {
     await exec(
-      `npx lerna version --conventional-commits --conventional-graduate --yes`
+      `npx lerna version --conventional-commits --conventional-graduate=${(await getPackagesToBump()).join(',')} --yes`
```

Comment:
Currently, this would run:
```bash
npx lerna version --conventional-commits --conventional-graduate=@coveo/atomic,@coveo/bueno,@coveo/headless,@coveo/quantic,coveo-atomic-example-070-alpha33,headless-react --yes
```

Reply:
Is the `--conventional-graduate` flag still needed if there are no alpha versions?

```
"Graduating" a package means bumping to the non-prerelease variant of a prerelease version,
eg. package-1@1.0.0-alpha.0 => package-1@1.0.0.
```


Ref: https://github.com/lerna/lerna/tree/main/commands/version#--conventional-graduate

Reply:
Also, I'm wondering if there is a need to version the `coveo-atomic-example` and `headless-react` packages since they are not published?

Maybe the lerna.json could just use one asterix:

```
"packages": [
    "packages/*"
  ],
```

Reply:
I don't think so, I'll exclude the samples.